api/src/main/java/org/apache/iceberg/ManifestFile.java

```diff
@@ -203,8 +203,12 @@ default boolean hasDeletedFiles() {
     /**
      * Returns true if at least one data file in the manifest has a nan value for the field.
      * Null if this information doesn't exist.
+     * <p>
```

Comment:
Nit: do should we add a closing </p> in the docstring?

Reply:
According to [this](https://www.oracle.com/technical-resources/articles/java/javadoc-tool.html#format) I think `<p>` itself might be enough?

> If you have more than one paragraph in the doc comment, separate the paragraphs with a \<p\> paragraph tag, as shown